server/router.js

```diff
@@ -5,9 +6,42 @@ const errors    = require('../api').errors;
 const respond   = require('./response-handler.js');
 const version   = require('./version.js');
 
+const ROUTE_MAP = {
+  GET: {
+    '/uuid': api.info.uuid,
+    '/server/connected': api.info.isConnected,
+    '/transaction-fee': api.info.fee,
+    '/server': api.info.serverStatus,
+    '/wallet/new': api.wallet.generate,
+    '/accounts/:account/payments': api.payments.getAccountPayments,
+    '/accounts/:account/payments/:identifier': api.payments.get,
+    '/accounts/:account/payments/paths/:destination_account/:destination_amount_string': api.payments.getPathFind,
+    '/accounts/:account/orders': api.orders.getOrders,
+    '/accounts/:account/order_book/:base/:counter': api.orders.getOrderBook,
+    '/accounts/:account/orders/:identifier': api.orders.getOrder,
+    '/accounts/:account/notifications': api.notifications.getNotification,
+    '/accounts/:account/notifications/:identifier':
+      api.notifications.getNotification,
+    '/accounts/:account/balances': api.balances.get,
+    '/accounts/:account/settings': api.settings.get,
+    '/transactions/:identifier': api.transactions.get,
+    '/accounts/:account/trustlines': api.trustlines.get
+  },
+  POST: {
+    '/accounts/:account/payments': api.payments.submit,
+    '/accounts/:account/orders': api.orders.placeOrder,
+    '/accounts/:account/settings': api.settings.change,
+    '/accounts/:account/trustlines': api.trustlines.add
+  },
+  DELETE: {
+    '/accounts/:account/orders/:sequence': api.orders.cancelOrder
+  }
+};
+
 var router = new express.Router();
+router.get('/', generateIndexPage);
 
-router.generateIndexPage = function(req, res) {
+function generateIndexPage(req, res) {
   var url_base = '/v' + version.getApiVersion();
 
   res.json({
```

Comment:
This isn't part of your changes, but since I'm not noticing it while looking at the diff. Responses should only be handled by the response handler.